src/Base/FileInfo.cpp

```diff
@@ -431,9 +431,7 @@ bool FileInfo::isDir() const
         return ((st.st_mode & _S_IFDIR) != 0);
 
 #elif defined(FC_OS_LINUX) || defined(FC_OS_CYGWIN) || defined(FC_OS_MACOSX) || defined(FC_OS_BSD)
-        struct stat st
-        {
-        };
+        struct stat st {};

```

Comment:
Could we just leave such changes out of scope? I guess something changed in the configuration as last time I was working on this file (to use std::filesystem), pre-commit insisted on the format it is changing from now (effectively doing the opposite)